### PR TITLE
Remove and Deprecate

### PR DESCRIPTION
**Pull request by bordoni**
_Ref:_ [C#43595](https://central.tri.be/issues/43595)

### CODE REVIEW

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Comment refers to a TODO task | [Main.php:267](https://github.com/moderntribe/event-tickets/blob/874a92c39b18ea6b5067b30de266477b6e25fe59/src/Tribe/Main.php#L267)

> via jenkins/codesniffer-phpcs

File: src/Tribe/Main.php [hunk 1]
```diff
@@ -256,48 +255,20 @@ public function hooks() {
...
-			'availability' => 'http://schema.org/' . in_array( true, $in_stock ) ? 'InStock' : 'OutOfStock',
-			'availabilityStarts' => get_gmt_from_date( date( Tribe__Date_Utils::DBDATETIMEFORMAT, min( $startdates ) ), 'c' ),
-			'availabilityEnds' => get_gmt_from_date( date( Tribe__Date_Utils::DBDATETIMEFORMAT, max( $enddates ) ), 'c' ),
-		);
-
-		$data->offers = $offer;
+		/**
+		 * @todo remove this after 4.4
+		 */
+		_deprecated_function( __CLASS__ . '::' . __METHOD__, '4.2', 'Tribe__Tickets__JSON_LD__Order' );
```
Comment (borkweb) [inline src/Tribe/Main.php]: The first parameter would become: `Tribe__Event__Tickets::Tribe__Event__Tickets::inject_tickets_json_ld`

Using `__METHOD__` all by itself should do what you are looking to do.

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Comment refers to a TODO task | [Main.php:267](https://github.com/moderntribe/event-tickets/blob/874a92c39b18ea6b5067b30de266477b6e25fe59/src/Tribe/Main.php#L267)

> via jenkins/codesniffer-phpcs

**Comment by borkweb** [comment]
:+1:

**Pull request merged**